dataset_configs/kazakh/ksc2/config.yaml

```diff
@@ -0,0 +1,130 @@
+documentation: |
+  Kazakh Speech Corpus 2
```

Comment:
In doc build, there is "WARNING: Title underline too short.", and some other warnings.
Please make sure there are no warnings.

Reply:
done